src/views/Home.jsx

```diff
@@ -18,12 +22,51 @@ export function Home({ listToken, setListToken }) {
 		}
 	}, [listToken]);
 
+	const handleChange = (e) => {
+		setExistingToken(e.target.value);
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
```

Comment:
Whenever a user creates a new token, it is always in lowercase. Perhaps we can handle case insensitivity by converting `existingToken` to all lowercase. 
```suggestion
const caseSensitiveToken = existingToken.toLowerCase()
```